README.md

```diff
@@ -21,11 +26,11 @@ module "cert" {
 
 ## Variables
 
-- `domain_name` - Primary domain name associated with certificate
-- `subject_alternative_names` - Subject alternative domain names
-- `hosted_zone_id` - Route 53 hosted zone ID for `domain_name`
-- `validation_record_ttl` - Route 53 record time-to-live (TTL) for validation record (default: `60`)
+-   `domain_name` - Primary domain name associated with certificate
+-   `subject_alternative_names` - Subject alternative domain names
+-   `hosted_zone_id` - Route 53 hosted zone ID for `domain_name`
+-   `validation_record_ttl` - Route 53 record time-to-live (TTL) for validation record (default: `60`)
```

Comment:
The `acm_certificate_tags` variable is missing from here. Additionally, could we use `tags` as a variable name instead?